prefab/curl-ssl/app/build.gradle

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright (C) 2019 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+apply plugin: 'com.android.application'
+apply plugin: 'kotlin-android'
+apply plugin: 'kotlin-android-extensions'
+
+android {
+    compileSdkVersion 29
+    defaultConfig {
+        applicationId "com.example.curlssl"
+        minSdkVersion 21
+        targetSdkVersion 29
+        versionCode 1
+        versionName "1.0"
+        testInstrumentationRunner "android.support.test.runner.AndroidJUnitRunner"
+
+        externalNativeBuild {
+            if (!project.hasProperty("ndkBuild")) {
+                cmake {
+                    arguments "-DANDROID_STL=c++_shared"
+                }
+            }
+        }
+    }
+
+    buildTypes {
+        release {
+            minifyEnabled false
+            proguardFiles getDefaultProguardFile('proguard-android-optimize.txt'), 'proguard-rules.pro'
+        }
+    }
+
+    externalNativeBuild {
+        if (project.hasProperty("ndkBuild")) {
+            ndkBuild {
+                path "src/main/cpp/Android.mk"
+            }
+        } else {
+            cmake {
+                path "src/main/cpp/CMakeLists.txt"
+            }
+        }
+    }
+}
+
+dependencies {
+    implementation fileTree(dir: 'libs', include: ['*.jar'])
+    implementation"org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version"
+    implementation 'com.android.support:appcompat-v7:29.0.0'
+    implementation 'com.android.support.constraint:constraint-layout:1.1.3'
+    // NB: These are not published to Maven yet. You can build them yourself with
+    // https://cs.android.com/android/platform/superproject/+/master:tools/ndkports/.
+    implementation "com.android.ndk.thirdparty:curl:7.66.0"
+    implementation "com.android.ndk.thirdparty:jsoncpp:1.9.1"
+    implementation "com.android.ndk.thirdparty:openssl:1.1.1d"
```

Comment:
Hi,
is there any progress on this? Is there a way to help get these packages published?

Reply:
Right now we're waiting for all the process in our end to resolve. There isn't a version of the Gradle plugin capable of using these yet either. Hopefully by the end of the month we'll have both of those available.

Reply:
They're published now. The latest version of this PR has the correct package names and versions.